packages/frontend/app/components/ui/chart.tsx

```diff
@@ -0,0 +1,400 @@
+'use client'
+
+import * as React from 'react'
+import * as RechartsPrimitive from 'recharts'
+import { cn } from '@/app/lib/utils'
+
```

Comment:
maybe add a comment mentioning that this is from shadcn plus mention if you've changed anything about it?

Reply:
addressed, added notes (only 1 small change)